client/src/components/ProfileSettings/EditProfile/EditProfileForm/EditProfileForm.tsx

```diff
@@ -0,0 +1,353 @@
+    setDateOfBirth(date);
+  };
+
+  const phoneNumberRegExp = /^\s*(?:\+?(\d{1,3}))?[-. (]*(\d{3})[-. )]*(\d{3})[-. ]*(\d{4})(?: *x(\d+))?\s*$/;
+
+  return (
+    <Formik
+      initialValues={{
+        type: ProfileType.Sitter,
+        firstName: '',
+        lastName: '',
+        gender: '',
+        dateOfBirth: new Date(),
+        email: '',
+        phoneNumber: '',
+        address: '',
+        description: '',
+        isAvailable: true,
+        availableHoursPerWeek: '',
+        hourlyRate: 1,
+      }}
+      validationSchema={Yup.object().shape({
+        firstName: Yup.string().max(15, 'First name must be 15 or less characters').required('Required'),
+        lastName: Yup.string().max(20, 'Last name must be 20 characters or less').required('Required'),
+        gender: Yup.string().required('Required'),
+        dateofBirth: Yup.date().required('Required'),
+        email: Yup.string().required('Requried').email('Invalid email address'),
+        phoneNumber: Yup.string()
+          .required('Required')
+          .matches(phoneNumberRegExp, 'Phone number is not valid')
+          .max(14, 'You must enter a ten-digit phone number with the area code')
+          .min(10, 'You must enter a ten-digit phone number with the area code'),
+        address: Yup.string().required('Required'),
+        description: Yup.string()
+          .max(200, 'Description must be 200 for less characters')
+          .min(1, 'Please describe yourself')
+          .required('Required'),
+        isAvailable: Yup.boolean().required('Required'),
+        availableHoursPerWeek: Yup.string().required('Required'),
+        hourlyRate: Yup.number()
+          .test('This is a valid rate', 'This is not a valid rate', (value) =>
+            value !== undefined ? /^\d+(?:\.\d{1,2})?$/.test(value.toString()) : false,
+          )
+          .max(200, 'The maximum you can charge is $200/hour')
+          .min(1, 'The minimum you can charge is $1/hour')
+          .required('Required'),
+      })}
+      onSubmit={handleSubmit}
+    >
+      {({ handleSubmit, handleChange, values, touched, errors, isSubmitting }) => (
+        <form onSubmit={handleSubmit} className={classes.form} noValidate>
+          <Grid className={classes.gridItemB}>
+            <Typography>
+              <h1 className={classes.heading}>Edit Profile</h1>
+            </Typography>
+          </Grid>
+          {values.type === ProfileType.Sitter && ( //TODO: Fix the button styling
+            <Grid className={`${classes.gridItemA} ${classes.toggle}`}>
+              <label className={classes.label}>{`I'M AVAILABLE`}</label>
+              <AvailableSwitch id="I'M AVAILABLE" checked={isAvailable} onChange={handleToggle} name="I'M AVAILABLE" />
+            </Grid>
+          )}
+          {values.type === ProfileType.Sitter && (
+            <Grid className={classes.gridItemA}>
+              <label className={classes.label}>AVAILABILITY</label>
```

Comment:
Reserve usage of capital case letters for abbreviations, you can change the text's case using CSS classes here instead. (Ditto throughout). Please see below for a reference on why to avoid capital casing:

Ref: https://medium.com/@mandy.michael/why-you-shouldnt-write-your-content-in-uppercase-instead-use-css-b03ac2c65b99